gamification/management/commands/update_participants_data.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+
+from gamification.process.update import (
+    get_mr_objects,
+    update_participants_data_with_mr,
+    get_issue_objects,
+    update_participants_data_with_issue,
+    get_participant_objects,
+    award_badges,
+)
+
+
+class Command(BaseCommand):
+    help = 'Update participants data'
+
+    def handle(self, *args, **options):
+        for mr in get_mr_objects():
+            if mr.labels.count() == 0:
```

Comment:
`if not mr.labels.count():`  (or maybe `if not mr.labels:` ? im not sure if django supports that) 

and this must be logged as a warning in our logging system.

someones work wasnt counted, and was silently discarded!!!

likewise with issues below.